code/datums/materials/basemats.dm

```diff
@@ -248,3 +248,143 @@ Unless you know what you're doing, only use the first three numbers. They're in
+	beauty_modifier = 0.25
+	turf_sound_override = FOOTSTEP_SAND
+	texture_layer_icon_state = "sand"
+
+//And now for our lavaland dwelling friends, sand, but in stone form! Truly revolutionary.
+/datum/material/sandstone
+	name = "sandstone"
+	id = "sandstone"
+	desc = "Bialtaakid 'ant taerif ma hdha."
+	color = "#B77D31"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/sandstone
+	value_per_unit = 0.0025
+	armor_modifiers = list("melee" = 0.5, "bullet" = 0.5, "laser" = 1.25, "energy" = 0.5, "bomb" = 0.5, "bio" = 0.25, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
+	beauty_modifier = 0.3
+	turf_sound_override = FOOTSTEP_WOOD
+	texture_layer_icon_state = "brick"
+
+/datum/material/snow
+	name = "snow"
+	id = "snow"
+	desc = "There's no business like snow business."
+	color = "#FFFFFF"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/mineral/snow
+	value_per_unit = 0.0025
+	armor_modifiers = list("melee" = 0.25, "bullet" = 0.25, "laser" = 0.25, "energy" = 0.25, "bomb" = 0.25, "bio" = 0.25, "rad" = 1.5, "fire" = 0.25, "acid" = 1.5)
+	beauty_modifier = 0.3
+	turf_sound_override = FOOTSTEP_SAND
+	texture_layer_icon_state = "sand"
+
+/datum/material/runedmetal
+	name = "runed metal"
+	id = "runed metal"
+	desc = "Mir'ntrath barhah Nar'sie."
+	color = "#3C3434"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/runed_metal
+	value_per_unit = 0.75
+	armor_modifiers = list("melee" = 1.2, "bullet" = 1.2, "laser" = 1, "energy" = 1, "bomb" = 1.2, "bio" = 1.2, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
+	beauty_modifier = -0.15
+	texture_layer_icon_state = "runed"
+
+/datum/material/bronze
+	name = "bronze"
+	id = "bronze"
+	desc = "Clock Cult? Never heard of it."
+	color = "#92661A"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/tile/bronze
+	value_per_unit = 0.025
+	armor_modifiers = list("melee" = 1, "bullet" = 1, "laser" = 1, "energy" = 1, "bomb" = 1, "bio" = 1, "rad" = 1.5, "fire" = 1.5, "acid" = 1.5)
+	beauty_modifier = 0.2
+
+/datum/material/paper
+	name = "paper"
+	id = "paper"
+	desc = "Ten thousand folds of pure starchy power."
+	color = "#E5DCD5"
+	categories = list(MAT_CATEGORY_RIGID = TRUE)
+	sheet_type = /obj/item/stack/sheet/paperframes
+	value_per_unit = 0.0025
+	armor_modifiers = list("melee" = 0.1, "bullet" = 0.1, "laser" = 0.1, "energy" = 0.1, "bomb" = 0.1, "bio" = 0.1, "rad" = 1.5, "fire" = 0, "acid" = 1.5)
+	beauty_modifier = 0.3
+	turf_sound_override = FOOTSTEP_SAND
+	texture_layer_icon_state = "paper"
+
+/datum/material/paper/on_applied_obj(obj/source, amount, material_flags)
+	. = ..()
+	if(material_flags & MATERIAL_AFFECT_STATISTICS)
+		var/obj/paper = source
+		paper.resistance_flags |= FLAMMABLE
+		paper.obj_flags |= UNIQUE_RENAME
+
+/datum/material/paper/on_removed_obj(obj/source, material_flags)
+	. = ..()
```

Comment:
Put this at the end as `return ..()`

Reply:
why

Reply:
For the same reason we use `. = ..()` at the start on `Initialize()` and `return ..()` at the end on `Destroy()`: because as a general rule we want to wait for things to set up on systems initializing, and to remove ourselves early on systems shutting down.

Reply:
that would be true for del() but this is a "on_x" type event which usually happen after a certain event and in which the order theoretically shouldn't matter in the same way as del()

Reply:
It's true for several of this class of events, such as  `on_mob_metabolize()` vs `on_mob_end_metabolize()`, `pickup()` vs `dropped()`, and such.
It's not followed in several places, that is true.
I think it's a good pattern to follow, however.
You can disregard it if you think it's better not.